static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss

```diff
@@ -57,10 +57,12 @@
     }
 }
 
-.paidfor-box {
+.paidfor-box--header,
```

Comment:
And in totally ideal world we shouldn't even create new popup styles as we already have popup 
![screen shot 2016-02-08 at 14 08 45](https://cloud.githubusercontent.com/assets/2579465/12887642/9b6bfca6-ce6d-11e5-9528-e770387eefee.png)

We should reuse that, add modifications (bg color) and some elements.